src/components/inputs/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,34 @@
+import { Text, TextInput, View } from 'react-native';
+import React, { useState } from 'react';
+import { useStyles } from 'react-native-unistyles';
+
+import { styleSheet } from './FormInput.style';
+
+type Props = {
```

Comment:
`FormINputProps`. Please try to avoid using default `Props` naming for the type, since it would be much harder to find this specific prop in you project in future, because you will have 50-100 types named the same